xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild

```diff
@@ -63,6 +63,15 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.19.1-gmodule-libs.patch #913864
```

Comment:
Don't use bare bug numbers, or trailing bash comments.

Reply:
Done.

Although this is a very common pattern which I copied from other ebuilds when I first started trying to submit patches to Gentoo and have never been told as being wrong before.